javascript.js

```diff
@@ -1 +1,57 @@
-console.log ('im working :) ')
\ No newline at end of file
+const books = [];
```

Comment:
- [ ] Please make sure the list is preserved in the local storage and reloaded into the page when it starts, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Make%20sure%20that%20data%20is%20preserved%20in%20the%20browser%27s%20memory%20by%20using%20localStorage.) requirement.